docs/sphinx/source/changelog/v2.1.2.rst

```diff
@@ -0,0 +1,15 @@
+**************************
+v2.1.2 (December 20, 2021)
+**************************
+
+Documentation
+-------------
+* Preferred citation updated (:issue:`296`, :pull:`300`)
+
+Bug fixes
+---------
+* XGBoost model saved in a version-stable json. Removes the xgboost version restriction of version 2.1.1. (:issue:`301`, :pull:`304`)
+
+Requirements
+------------
+* ``joblib`` removed as requirement (:pull:`304`)
```

Comment:
In the past we've noted changes to `requirements.txt` here, may be good to do the same for `xgboost`